invokeai/app/invocations/flux_text_to_image.py

```diff
@@ -108,6 +112,35 @@ def _run_diffusion(
         with transformer_info as transformer:
             assert isinstance(transformer, Flux)
 
+            def step_callback(img: torch.Tensor, state: PipelineIntermediateState) -> None:
+                if context.util.is_canceled():
+                    raise CanceledException
+
+                # TODO: Make this look like the image
+                latent_image = unpack(img.float(), self.height, self.width)
+                latent_image = latent_image.squeeze()  # Remove unnecessary dimensions
+                flattened_tensor = latent_image.reshape(-1)  # Flatten to shape [48*128*128]
+
+                # Create a new tensor of the required shape [255, 255, 3]
+                latent_image = flattened_tensor[: 255 * 255 * 3].reshape(255, 255, 3)  # Reshape to RGB format
+
+                # Convert to a NumPy array and then to a PIL Image
+                image = Image.fromarray(latent_image.cpu().numpy().astype(np.uint8))
+
+                (width, height) = image.size
+                width *= 8
+                height *= 8
+
+                dataURL = image_to_dataURL(image, image_format="JPEG")
+
+                # TODO: move this whole function to invocation context to properly reference these variables
```

Comment:
I for a long time planning to rewrite this part, so that event accepts not latents, but image.
As there looks no progress, I think I can create this PR separately and it will make this moment less problematic.

Reply:
Sounds good, thanks so much for catching this and reaching out!